internal/cmd/gocdk/internal/static/_assets/launchers/ecs.tf

```diff
@@ -0,0 +1,170 @@
+# Create an ECR Docker repository.
+# The name used here will be part of the deployed Docker image name.
+resource "aws_ecr_repository" "default" {
+  name = "{{.ProjectName}}"
+}
+
+# ECS Cluster
```

Comment:
WDYT about a comment here saying something like "The resources below are needed to create a working ECS cluster. The AWS UI creates them for you automatically, but the `aws` CLI does not." ?